GameData/RealismOverhaul/RO_SuggestedMods/Bluedog_DB/RO_BDB_RealPlume/BDB_Atlas_Classic.cfg

```diff
@@ -81,7 +81,6 @@
             fixedEmissions = false
             sizeClamp = 50
             randomInitalVelocityOffsetMaxRadius = 0
-            randConeEmit
```

Comment:
Someone who understands plume code (@al2me6 ?) needs to look at these, I'm not sure it's correct to nuke the value rather than do something with it

Reply:
Hm I didn't get notified for some reason. Should be fine to just nuke.